src/main.ts

```diff
@@ -113,24 +113,30 @@ async function main() {
 
     console.log("Deployment Step Started");
 
-    let containerAppDeploymentResult = await client.containerApps.beginCreateOrUpdateAndWait(
+    // update
+    await client.containerApps.beginUpdateAndWait(
       taskParams.resourceGroup,
       taskParams.containerAppName,
       containerAppEnvelope,
     );
 
-    if (containerAppDeploymentResult.provisioningState == "Succeeded") {
-      console.log("Deployment Succeeded");
-
-      if (ingresConfig.external == true) {
-        let appUrl = "http://" + containerAppDeploymentResult.latestRevisionFqdn + "/"
-        core.setOutput("app-url", appUrl);
-        console.log("Your App has been deployed at: " + appUrl);
+    // check if added revision is included in revision list
```

Comment:
Do we need to check all the list of revisions? If we just want to check the duplicate revision, can't we use the Show API?

ex

```
if (revision.exists?(new_revision) == true)
	puts "Revision exists.";
else
	puts "Revision does not exist.";
end
```

Reply:
I concern an issue like #41. This issue is that adding revision is failed without any error/exception under certain confition (ex revision name length in case of #41). So, I added the code getting revision list as double-checking.

Reply:
I see. I checked an example of a revision without a name 😄

Reply:
Sorry, the issune number was wrong in the above comment. Fixed already.

Reply:
Changed check-login after updating